airbyte-cdk/python/pyproject.toml

```
@@ -114,3 +115,10 @@ optional_poetry_groups = ["dev"]
 poetry_extras = ["file-based", "sphinx-docs", "vector-db-based"]
 poe_tasks = ["check-ci"]
```

I would ideally recommend adding `poetry run code flash` as a poe task and add this poe task here
```suggestion
poe_tasks = ["check-ci", "codeflash"]
required_env_variables = ["CODEFLASH_API_KEY", "CODEFLASH_PR_NUMBER"]
```
This would prevent the introduction of a new GHA workflow and make codeflash run on the listed `python_versions`.

You'd need to set the env vars in the GHA workflow [here](https://github.com/airbytehq/airbyte/blob/master/.github/workflows/airbyte-ci-tests.yml#L47)